src/applications/vaos/tests/actions/newAppointment.unit.spec.js

```diff
@@ -674,7 +675,11 @@ describe('VAOS newAppointment actions', () => {
         },
       });
       setFetchJSONResponse(global.fetch.onCall(3), clinics);
-      setFetchJSONResponse(global.fetch.onCall(4), facilityDetails);
+      setFetchJSONResponse(global.fetch.onCall(4), pastAppointments);
```

Comment:
This is what was tripping me up.  I forgot that the long term history check actually performs 4 fetchs and this stubbing kept causing errors until I repeated them.